### PR TITLE
include hurricane example

### PR DESCRIPTION
**Pull request by longshuicy**

### CODE REVIEW

**Comment by ywkim312** [comment]
should I install pyincore branch or something to test this notebook?

**Comment by longshuicy** [comment]
Service change isn't deployed on production yet so the results don't make much sense yet

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Changed
 - Updated branch name in links [#137](https://github.com/IN-CORE/incore-docs/issues/137).
 
+### Added
+- Include Galveston Hurricane EPF damage example [#153](https://github.com/IN-CORE/incore-docs/issues/153)
+
+
```
Comment (nunezco2) [inline CHANGELOG.md:15]: Extra space

**Review by nunezco2 — CHANGES_REQUESTED** [review]

File: notebooks/epf_dmg.ipynb [hunk 2]
```diff
@@ -50,56 +57,87 @@
...
+    "fragility_service = FragilityService(client)\n",
+    "mapping_set = MappingSet(fragility_service.get_mapping(\"62fac92ecef2881193f22613\"))\n",
+    "epf_dmg_hurricane_galveston.set_input_dataset('dfr3_mapping_set', mapping_set)\n",
+    "\n",
+    "epf_dmg_hurricane_galveston.set_parameter(\"result_name\", \"galveston_hurricane_epf_damage\")\n",
+    "epf_dmg_hurricane_galveston.set_parameter(\"hazard_type\", \"hurricane\")\n",
+    "epf_dmg_hurricane_galveston.set_parameter(\"hazard_id\", \"5fa5a228b6429615aeea4410\")\n",
+    "epf_dmg_hurricane_galveston.set_parameter(\"fragility_key\", \"Non-Retrofit Fragility ID Code\")\n",
+    "epf_dmg_hurricane_galveston.set_parameter(\"num_cpu\", 8)\n",
+    "# Run Analysis\n",
```
Comment (ywkim312) [inline notebooks/epf_dmg.ipynb:124]: Maybe add a blank line above this?

**Review by ywkim312 — APPROVED** [review]

**Comment by navarroc** [comment]
@nunezco2 can you re-review this. I think your requested change was addressed. Is there anything else?

**Review by nunezco2 — APPROVED** [review]

**Comment by nunezco2** [comment]
Done. This should be ready by now.

**Pull request merged**